platform-sdk/swirlds-platform-core/src/test/java/com/swirlds/platform/state/signed/ReservedSignedStateTests.java

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright (C) 2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.swirlds.platform.state.signed;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertSame;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import com.swirlds.common.exceptions.ReferenceCountException;
+import com.swirlds.platform.state.RandomSignedStateGenerator;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+@DisplayName("ReservedSignedState Tests")
+class ReservedSignedStateTests {
+
+    @Test
+    @DisplayName("Null State Test")
+    void nullStateTest() {
+        try (final ReservedSignedState reservedSignedState = new ReservedSignedState()) {
+            assertThrows(NullPointerException.class, reservedSignedState::get);
```

Comment:
We specified that we want to use AssertJ for all tests (I know that from Services since there all tests are moving to AssertJ and as far as I know that is a global (platform + services) decision and not only services)